src/platform_storage_api/admission_controller/volume_resolver.py

```diff
@@ -0,0 +1,245 @@
+import asyncio
+import dataclasses
+import logging
+from asyncio import Task
+from enum import Enum
+from pathlib import PurePath
+from types import TracebackType
+from typing import Any, Optional
+from uuid import uuid4
+
+import aiorwlock
+from apolo_kube_client.client import KubeClient
+
+from platform_storage_api.storage import StoragePathResolver
+
+
+logger = logging.getLogger(__name__)
+
+KUBE_PLATFORM_STORAGE_APP_NAME = "platform-storage"
+KUBE_PLATFORM_NAMESPACE = "platform"
```

Comment:
Let's not use hardcoded namespace, platform services can be installed in any namespace, I think kube config already contains it

Reply:
removed that